projects/packages/account-protection/src/assets/jetpack-logo.php

```diff
@@ -0,0 +1,21 @@
+<svg
```

Comment:
that was probably supposed to be a .svg file? 😉

Reply:
Good eye!